src/wallet/rpcdump.cpp

```diff
@@ -102,6 +102,10 @@ UniValue importprivkey(const JSONRPCRequest& request)
         );
 
 
+    if (pwallet->IsWalletFlagSet(WALLET_FLAG_DISABLE_HOT_KEYS)) {
+        throw JSONRPCError(RPC_WALLET_ERROR, "Error: Hot keys are disabled (-disablehot)");
```

Comment:
It doesn't make sense to mention the option here, since it's tied to the wallet and can't just be toggled...